Vagrantfile

```diff
@@ -207,7 +253,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     if Vagrant.has_plugin?("vagrant-vbguest")
         config.vbguest.auto_update = false
     end 
-    if ENV['CONTIV_NODE_OS'] && ENV['CONTIV_NODE_OS'] == "ubuntu" then
+    if node_os == "ubuntu" then
```

Comment:
`String#%` taking a hash argument is really uncommon in Ruby code, and you can think of it being similar to Python's `"{foo} {bar}".format(foo="foo", bar="bar")` construct